app-backend/db/src/main/scala/com/azavea/rf/database/notification/templates/PlainGroupInvitation.scala

```diff
@@ -0,0 +1,41 @@
+package com.azavea.rf.database.notification.templates
+
+import com.azavea.rf.database._
+import com.azavea.rf.datamodel._
+
+import doobie.ConnectionIO
+
+import java.util.UUID
+
+case class PlainGroupInvitation(
+  groupId: UUID,
+  groupType: GroupType,
+  subjectId: String,
+  platformHost: String
+) {
+  def build: ConnectionIO[String] = {
+    for {
+      subjectEmail <- UserDao.unsafeGetUserById(subjectId).map((usr: User) => usr.email)
+      groupName <- groupType match {
+        case GroupType.Team =>
+          TeamDao.unsafeGetTeamById(groupId) map { (team: Team) => team.name }
+        case GroupType.Organization =>
+          OrganizationDao.unsafeGetOrganizationById(groupId) map {
+            (org: Organization) => org.name
+          }
+        case _ =>
+          throw new IllegalArgumentException(
+            "Users can only request to add themselves to organizations and teams"
+          )
+      }
+    } yield {
+      s"""
+      | User ${subjectEmail} has invited you to join their ${groupType.toString.toLowerCase},
+      | ${groupName}.
+      | To approve or reject this request, visit the following page:
+      | https://${platformHost}/jkjkjkjkjkjkjk
```

Comment:
@alkamin - could you also point us to the URL for this [page](https://app.goabstract.com/projects/cd34e180-d61d-11e7-b99b-5793a477ef34/branches/95c06d80-761f-11e8-bf41-df47385a2ce8/commits/32707e37862e20733287063a90de5b3b5ebed50c/files/B7A9D19F-97DB-441C-97ED-F692000DA14B/layers/A8BECD43-35AD-45E4-8B07-973C9D72409D?mode=design)

Reply:
```Subject: 
Join {Org_Name / Team_Name} on {Platform_Name}

Body:
{Requester_Name} has invited you to join the {Team / Organization} {Org_Name / Team_Name} on {Platform_Name}. To approve or reject this request, visit {Admin_Link}.

If you have questions, please feel free to reach out any time at *{Platform_Help_Email}*.

- The *{Platform_Name}* Team```